test/tests/transition-aborted-error_test.js

```diff
@@ -0,0 +1,20 @@
+import { module, test } from './test_helpers';
+import TransitionAbortedError from 'router/transition-aborted-error';
+
+module('transition-aborted-error');
+
+test('correct inheritance and name', function(assert) {
+  var error;
+
+  try {
+    throw new TransitionAbortedError('Message');
+  } catch(e) {
+    error = e;
+  }
+
+  // it would be more correct with TransitionAbortedError, but other libraries may rely on this name
+  assert.equal(error.name, 'TransitionAborted', "TransitionAbortedError has the name 'TransitionAborted'");
```

Comment:
I guess we cannot change the name, since other may rely on this, right?

https://github.com/travis-ci/travis-web/blob/ca5ef0044f3ee4e51c7cf4dd46b1e44e1cc980a8/app/services/raven.js#L7